app.js

```diff
@@ -14,7 +14,7 @@ function create(env, ctx) {
     var appInfo = env.name + ' ' + env.version;
     app.set('title', appInfo);
     app.enable('trust proxy'); // Allows req.secure test on heroku https connections.
-    if (!env.settings.isEnabled('insecureUseHttp')) {
+    if (!process.env.INSECURE_USE_HTTP=='true') {
```

Comment:
This is case sensitive, boolean checks should at least do case insensitive comparison